### ISSUE TITLE
VS Crash during Web Site Project creation

### ISSUE THREAD

**Issue by jmarolf** [dotnet/roslyn#8841]
1. Install latest from stabilization
2. Create ASP.NET  Web Site Project
3. Navigate to a code file

Expected: Web Stie is Created

Actual:  Visual Studio Crash

```
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.FatalError.Report(System.Exception exception, System.Action<System.Exception> handler) Line 151        C#
> Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.ProcessCompilationEventsCoreAsync(Microsoft.CodeAnalysis.Diagnostics.AnalysisScope analysisScope, Microsoft.CodeAnalysis.Diagnostics.AnalysisState analysisStateOpt, bool prePopulatedEventQueue, System.Threading.CancellationToken cancellationToken) Line 770              C#
               [Resuming Async Method]           
               [Native to Managed Transition] 
   System.Collections.Immutable.dll!System.Collections.Immutable.ImmutableArray<System.__Canon>.this[int].get(int index)   Unknown
Microsoft.CodeAnalysis.CSharp.dll!Microsoft.CodeAnalysis.CSharp.BoundCall.DeriveArgument.AnonymousMethod__0(Microsoft.CodeAnalysis.CSharp.BoundExpression argument) Line 174          C#
mscorlib.dll!System.Runtime.CompilerServices.ConditionalWeakTable<Microsoft.CodeAnalysis.CSharp.BoundExpression, Microsoft.CodeAnalysis.Semantics.IArgument>.GetValue(Microsoft.CodeAnalysis.CSharp.BoundExpression key, System.Runtime.CompilerServices.ConditionalWeakTable<Microsoft.CodeAnalysis.CSharp.BoundExpression, Microsoft.CodeAnalysis.Semantics.IArgument>.CreateValueCallback createValueCallback)             Unknown
               Microsoft.CodeAnalysis.CSharp.dll!Microsoft.CodeAnalysis.CSharp.BoundCall.DeriveArgument(int parameterIndex, int argumentIndex, System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.CSharp.BoundExpression> boundArguments, System.Collections.Immutable.ImmutableArray<string> argumentNames, System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.RefKind> argumentRefKinds, System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.CSharp.Symbols.ParameterSymbol> parameters, Microsoft.CodeAnalysis.SyntaxNode invocationSyntax) Line 147        C#
Microsoft.CodeAnalysis.CSharp.dll!Microsoft.CodeAnalysis.CSharp.BoundCall.Microsoft.CodeAnalysis.Semantics.IInvocationExpression.ArgumentsInSourceOrder.get() Line 54  C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Semantics.OperationWalker.VisitInvocationExpression(Microsoft.CodeAnalysis.Semantics.IInvocationExpression operation) Line 196         C#
Microsoft.CodeAnalysis.CSharp.dll!Microsoft.CodeAnalysis.CSharp.BoundCall.Accept(Microsoft.CodeAnalysis.Semantics.OperationVisitor visitor) Line 77   C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Semantics.OperationWalker.Visit(Microsoft.CodeAnalysis.IOperation operation) Line 34            C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Semantics.OperationExtensions.OperationCollector.Visit(Microsoft.CodeAnalysis.IOperation operation) Line 64            C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Semantics.OperationWalker.VisitExpressionStatement(Microsoft.CodeAnalysis.Semantics.IExpressionStatement operation) Line 178         C#
Microsoft.CodeAnalysis.CSharp.dll!Microsoft.CodeAnalysis.CSharp.BoundExpressionStatement.Accept(Microsoft.CodeAnalysis.Semantics.OperationVisitor visitor) Line 699 C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Semantics.OperationWalker.Visit(Microsoft.CodeAnalysis.IOperation operation) Line 34            C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Semantics.OperationExtensions.OperationCollector.Visit(Microsoft.CodeAnalysis.IOperation operation) Line 64            C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Semantics.OperationWalker.VisitArray<Microsoft.CodeAnalysis.IOperation>(System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.IOperation> list) Line 21          C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Semantics.OperationWalker.VisitBlockStatement(Microsoft.CodeAnalysis.Semantics.IBlockStatement operation) Line 45            C#
Microsoft.CodeAnalysis.CSharp.dll!Microsoft.CodeAnalysis.CSharp.BoundBlock.Accept(Microsoft.CodeAnalysis.Semantics.OperationVisitor visitor) Line 52   C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Semantics.OperationWalker.Visit(Microsoft.CodeAnalysis.IOperation operation) Line 34            C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Semantics.OperationExtensions.OperationCollector.Visit(Microsoft.CodeAnalysis.IOperation operation) Line 64            C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Semantics.OperationExtensions.DescendantsAndSelf(Microsoft.CodeAnalysis.IOperation operation) Line 32            C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver<Microsoft.CodeAnalysis.CSharp.SyntaxKind>.GetOperationsToAnalyze(System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.IOperation> operationBlocks) Line 1777           C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver<Microsoft.CodeAnalysis.CSharp.SyntaxKind>.ExecuteDeclaringReferenceActions(Microsoft.CodeAnalysis.SyntaxReference decl, int declarationIndex, Microsoft.CodeAnalysis.Diagnostics.SymbolDeclaredCompilationEvent symbolEvent, Microsoft.CodeAnalysis.Diagnostics.AnalysisScope analysisScope, Microsoft.CodeAnalysis.Diagnostics.AnalysisState analysisStateOpt, bool shouldExecuteSyntaxNodeActions, bool shouldExecuteOperationActions, bool shouldExecuteCodeBlockActions, bool shouldExecuteOperationBlockActions, bool isInGeneratedCode, System.Threading.CancellationToken cancellationToken) Line 1542           C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver<Microsoft.CodeAnalysis.CSharp.SyntaxKind>.ExecuteDeclaringReferenceActions(Microsoft.CodeAnalysis.Diagnostics.SymbolDeclaredCompilationEvent symbolEvent, Microsoft.CodeAnalysis.Diagnostics.AnalysisScope analysisScope, Microsoft.CodeAnalysis.Diagnostics.AnalysisState analysisStateOpt, bool isGeneratedCodeSymbol, System.Threading.CancellationToken cancellationToken) Line 1401           C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.ProcessSymbolDeclared(Microsoft.CodeAnalysis.Diagnostics.SymbolDeclaredCompilationEvent symbolEvent, Microsoft.CodeAnalysis.Diagnostics.AnalysisScope analysisScope, Microsoft.CodeAnalysis.Diagnostics.AnalysisState analysisStateOpt, System.Threading.CancellationToken cancellationToken) Line 831         C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.ProcessEventCore(Microsoft.CodeAnalysis.Diagnostics.CompilationEvent e, Microsoft.CodeAnalysis.Diagnostics.AnalysisScope analysisScope, Microsoft.CodeAnalysis.Diagnostics.AnalysisState analysisStateOpt, System.Threading.CancellationToken cancellationToken) Line 789          C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.ProcessEvent(Microsoft.CodeAnalysis.Diagnostics.CompilationEvent e, Microsoft.CodeAnalysis.Diagnostics.AnalysisScope analysisScope, Microsoft.CodeAnalysis.Diagnostics.AnalysisState analysisStateOpt, System.Threading.CancellationToken cancellationToken) Line 778          C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.ProcessCompilationEventsCoreAsync(Microsoft.CodeAnalysis.Diagnostics.AnalysisScope analysisScope, Microsoft.CodeAnalysis.Diagnostics.AnalysisState analysisStateOpt, bool prePopulatedEventQueue, System.Threading.CancellationToken cancellationToken) Line 765              C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.ProcessCompilationEventsAsync(Microsoft.CodeAnalysis.Diagnostics.AnalysisScope analysisScope, Microsoft.CodeAnalysis.Diagnostics.AnalysisState analysisStateOpt, bool prePopulatedEventQueue, System.Threading.CancellationToken cancellationToken) Line 700              C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.ExecutePrimaryAnalysisTaskAsync(Microsoft.CodeAnalysis.Diagnostics.AnalysisScope analysisScope, Microsoft.CodeAnalysis.Diagnostics.AnalysisState analysisStateOpt, bool usingPrePopulatedEventQueue, System.Threading.CancellationToken cancellationToken) Line 359             C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver.AttachQueueAndProcessAllEventsAsync(Microsoft.CodeAnalysis.Diagnostics.AsyncQueue<Microsoft.CodeAnalysis.Diagnostics.CompilationEvent> eventQueue, Microsoft.CodeAnalysis.Diagnostics.AnalysisScope analysisScope, Microsoft.CodeAnalysis.Diagnostics.AnalysisState analysisStateOpt, System.Threading.CancellationToken cancellationToken) Line 288         C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Diagnostics.CompilationWithAnalyzers.ComputeAnalyzerDiagnosticsCoreAsync(Microsoft.CodeAnalysis.Diagnostics.AnalyzerDriver driver, Microsoft.CodeAnalysis.Diagnostics.AsyncQueue<Microsoft.CodeAnalysis.Diagnostics.CompilationEvent> eventQueue, Microsoft.CodeAnalysis.Diagnostics.AnalysisScope analysisScope, System.Threading.CancellationToken cancellationToken) Line 685         C#
Microsoft.CodeAnalysis.dll!Microsoft.CodeAnalysis.Diagnostics.CompilationWithAnalyzers.ComputeAnalyzerDiagnosticsAsync.AnonymousMethod__1() Line 575                C#
               mscorlib.dll!System.Threading.Tasks.Task<System.Threading.Tasks.Task>.InnerInvoke()  Unknown
               mscorlib.dll!System.Threading.Tasks.Task.Execute()         Unknown
               mscorlib.dll!System.Threading.Tasks.Task.ExecutionContextCallback(object obj) Unknown
               mscorlib.dll!System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, object state, bool preserveSyncCtx)           Unknown
               mscorlib.dll!System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, object state, bool preserveSyncCtx) Unknown
               mscorlib.dll!System.Threading.Tasks.Task.ExecuteWithThreadLocal(ref System.Threading.Tasks.Task currentTaskSlot)               Unknown
               mscorlib.dll!System.Threading.Tasks.Task.ExecuteEntry(bool bPreventDoubleExecution) Unknown
               mscorlib.dll!System.Threading.Tasks.Task.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()                Unknown
               mscorlib.dll!System.Threading.ThreadPoolWorkQueue.Dispatch()             Unknown
               mscorlib.dll!System.Threading._ThreadPoolWaitCallback.PerformWaitCallback() Unknown

```

**Comment by davkean** [comment]
@jmarolf Can you verify that you don't have any analyzers installed and try again? Just trying to understand the impact here.

**Comment by jmarolf** [comment]
Looks like I had some analyzers loaded in VS from a previous solution.  After closing the solution and running install dogfood, I don't see the crash

**Comment by JohnHamby** [comment]
Fixed by #8862
